pallets/utxo/src/lib.rs

```diff
@@ -562,6 +677,58 @@ pub mod pallet {
         Ok(().into())
     }
 
+    pub fn tokens_create<T: Config>(
+        caller: &T::AccountId,
+        public: H256,
+        input_for_fee: TransactionInput,
+        token_name: String,
+        token_ticker: String,
+        supply: Value,
+    ) -> DispatchResultWithPostInfo {
+        ensure!(!supply.is_zero(), Error::<T>::MinBalanceZero);
+
+        // Take a free TokenID
+        let token_id =
+            <TokensHigherID<T>>::get().checked_add(1).ok_or("All tokens IDs has taken")?;
+        sp_runtime::print("TOKEN ID IS");
+        sp_runtime::print(&token_id);
```

Comment:
It may be easier to use `log::info!` here. I don't know how `sp_runtime::print` works but it could be written like this instead: `log::info!("TOKEN ID IS {:?}", token_id);`

Reply:
Thank you for pointing, this print I used to debug, and I thought that I've already removed it. This is an unnecessary print here.

Reply:
Removed now